app/models/jellyfish_azure/service/azure_service.rb

```diff
@@ -0,0 +1,92 @@
+module JellyfishAzure
+  module Service
+    class AzureService < ::Service::Storage
```

Comment:
May want to use just use `::Service` here. `Storage` may add undesired changes in the future.